backend/codesirius/problems/models/sample_test.py

```
@@ -0,0 +1,14 @@
+
+from problems.models import Problem
+
+
+class SampleTest(models.Model):
+    problem = models.ForeignKey(
+        Problem, on_delete=models.CASCADE, related_name="sample_tests"
+    )
+    input = models.TextField(max_length=1000)
+    output = models.TextField(max_length=1000)
+
+    def __str__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__str__`

Add return type annotation: `str`

(ANN204)